pallets/deitos/src/lib.rs

```diff
@@ -91,20 +95,38 @@ pub mod pallet {
         Transfer,
     }
 
+    #[pallet::genesis_config]
+    #[derive(frame_support::DefaultNoBound)]
+    pub struct GenesisConfig<T: Config> {
+        /// Genesis Initial IP Bounding
+        pub initial_ip_bounding: BalanceOf<T>,
```

Comment:
correct, that defines in the chain spec what is the initial deposit required to become an IP in the system. The idea of of having this value in the storage item `IPBoundingAmount` so this value can be changed from a governance call instead of having it hardcoded in the runtime as a constant

Reply:
Agree, that's good to have it stored, but don't understand the naming. Shouldn't it be just `deposit` instead of `bounding`?